base/local/prometheus.yaml

```diff
@@ -209,4 +209,5 @@ spec:
       resources:
         requests:
           storage: 200Gi
+      storageClassName: ebs-gp2
```

Comment:
I remember this rubbish, it gets ugly with gcp, but I don't think there is a solution to it.

Reply:
To be fair we wouldn't need storage classes if we had default storage class set to ebs-gp2

https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/